src/Postman/CollectionWriter.php

```diff
@@ -37,14 +37,15 @@ public function getCollection()
                     'name' => $groupName,
                     'description' => '',
                     'item' => $routes->map(function ($route) {
+                        $mode = $route['methods'][0] === 'PUT' ? 'urlencoded' : 'formdata';
```

Comment:
StyleCI wants a newline before the return statement.

```suggestion
                        $mode = $route['methods'][0] === 'PUT' ? 'urlencoded' : 'formdata';

```